projects/storefrontlib/src/cms-components/myaccount/update-email/update-email.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import {
+  AuthService,
+  GlobalMessageService,
+  GlobalMessageType,
+  RoutingService,
+  UserService,
+} from '@spartacus/core';
+import { Observable, Subscription } from 'rxjs';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
```

Comment:
we also use automated imports in our IDE, which sorts imports automatically on save. very convenient and never any differences between devs!